server_adapter_go.go

```diff
@@ -0,0 +1,416 @@
+package revel
```

Comment:
there is already a file called `template_agapter_go.go` so this is consistent with that